tests/zero_code_change/tensorflow2_integration_tests.py

```diff
@@ -77,6 +80,103 @@ def test_keras_v2(script_mode: bool = False, eager_mode: bool = True):
         trial = smd.create_trial(path=sim.out_dir)
         assert len(trial.steps()) > 0, "Nothing saved at any step."
         assert len(trial.tensor_names()) > 0, "Tensors were not saved."
+        assert len(trial.tensor_names(collection="losses")) > 0
+
+
+def helper_test_keras_v2_json_config(
+    json_file_contents, script_mode: bool = False, eager_mode: bool = True
+):
+    """ Works as intended. """
+    smd.del_hook()
```

Comment:
How do these new tests differ from the existing tests that are already in the codebase? Might be better to just update the existing Keras tests.

Reply:
not very different. its a mix of existing tests. json config from tf 1.x tests and model from tf 2.x test. this is meant only for TF 2.x. we are testing eager and non-eager mode in TF 2.x.